cfg/asd_demo.json

```diff
@@ -1,16 +1,23 @@
 {
- "log_level":"debug",
- // should be an absolute path
- "home": "/tmp/asd_demo",
- "node_id": "xxxxxxxxxx",
- // optionally specify the ips to listen for incoming client connections
- // "ips" : ["1.2.3.4", "2.3.4.4"],
- "port": 10000,
- // optional asd_id is optional
- "asd_id" : "a globally unique id for the asd",
- // optional, we use 90% of the file system capacity
- "limit" : 90
- // turn off syncing of batches;
- // don't uncomment this unless you know what you're doing. warranty void
- //, "__sync_dont_use" : false
+    "log_level":"debug",
+    // should be an absolute path
+    "home": "/tmp/asd_demo",
+    "node_id": "xxxxxxxxxx",
+    // optionally specify the ips to listen for incoming client connections
+    // "ips" : ["1.2.3.4", "2.3.4.4"],
+    "port": 10000,
```

Comment:
document this is now optional, and that not setting it disable plain socket access